File: ApiSuvesaPos/Negocio/Modelo/caja/aperturacaja.vb
Namespace Modelo
    Public Class aperturacaja
        Public Property NApertura As Integer
        Public Property Fecha As Date
        Public Property Nombre As String
        Public Property Estado As String
        Public Property Observaciones As String
        Public Property Anulado As Boolean
        Public Property Cedula As String
        Public Property Num_Caja As Long
        Public Property Apertura_Total_Tope As New Modelo.apertura_total_tope
        Public Property apertura_denominaciones As New List(Of Modelo.apertura_denominaciones)
    End Class

    Public Class apertura_total_tope
        Public Property id_total_tope As Integer
        Public Property NApertura As Integer
        Public Property CodMoneda As Integer
        Public Property Monto_Tope As Decimal
        Public Property MonedaNombre As String
    End Class

    Public Class apertura_denominaciones
        Public Property Id As Long
        Public Property Id_Apertura As Integer
        Public Property Id_Denominacion As Integer
        Public Property Monto As Decimal
        Public Property Cantidad As Decimal
    End Class

End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/caja/aperturacaja.cs
using System;
using System.Collections.Generic;

namespace Modelo
{
    public class aperturacaja
    {
        public int NApertura { get; set; }
        public DateTime Fecha { get; set; }
        public string Nombre { get; set; }
        public string Estado { get; set; }
        public string Observaciones { get; set; }
        public bool Anulado { get; set; }
        public string Cedula { get; set; }
        public long Num_Caja { get; set; }
        public Modelo.apertura_total_tope Apertura_Total_Tope { get; set; } = new Modelo.apertura_total_tope();
        public List<Modelo.apertura_denominaciones> apertura_denominaciones { get; set; } = new List<Modelo.apertura_denominaciones>();
    }

    public class apertura_total_tope
    {
        public int id_total_tope { get; set; }
        public int NApertura { get; set; }
        public int CodMoneda { get; set; }
        public decimal Monto_Tope { get; set; }
        public string MonedaNombre { get; set; }
    }

    public class apertura_denominaciones
    {
        public long Id { get; set; }
        public int Id_Apertura { get; set; }
        public int Id_Denominacion { get; set; }
        public decimal Monto { get; set; }
        public decimal Cantidad { get; set; }
    }
}
